core/modules/xrdsvc/StreamBuffer.cc

```diff
@@ -38,17 +38,42 @@ LOG_LOGGER _log = LOG_GET("lsst.qserv.xrdsvc.StreamBuffer");
 }
 
 
+using namespace std;
+
+
 namespace lsst {
 namespace qserv {
 namespace xrdsvc {
 
-std::atomic<size_t> StreamBuffer::_totalBytes(0);
+
+atomic<int64_t> StreamBuffer::_maxTotalBytes{40'000'000'000};
+atomic<int64_t> StreamBuffer::_totalBytes(0);
+mutex StreamBuffer::_createMtx;
+condition_variable StreamBuffer::_createCv;
+
+
+static void StreamBuffer::setMaxTotalBytes(int64_t maxBytes) {
+    LOGS(_log, LOG_LVL_INFO, "StreamBuffer::setMaxTotalBytes maxBytes=" << maxBytes);
+    if (maxBytes < 0) {
+        throw invalid_argument("StreamBuffer::setMaxTotalBytes negative " + to_string(maxBytes));
+    }
+    if (maxBytes < 1.0e9) {
```

Comment:
Mixing integer and floating pointing comparison looks confusing (to me). Why not compare with?
```
1'000'000'000
```